# actions (rosrun or roslaunch) or system command:
string action

# ROS namespace where the program is executed
# temoto_namespace of the process manager if empty string is given
string ros_namespace

# name of the package
string package_name

# name of the node or launch file
string executable

# additional arguments
string args

temoto_resource_registrar/TemotoRequestMetadata temoto_metadata
---

# The process id of the loaded resource
int32 pid

temoto_resource_registrar/TemotoResponseMetadata temoto_metadata
